src/core/a-entity.js

```diff
@@ -244,35 +252,38 @@ var proto = Object.create(ANode.prototype, {
 
   /**
    * Initialize component.
+   *
+   * @param {string} name - the component name.
```

Comment:
nit: but could be more concise without `the`, and consistent caps

`Component name.` + `Component data.`